gavel/templates/admin_annotator.html

```diff
@@ -24,8 +24,8 @@
         <td>{{ annotator.description | safe }}</td>
       </tr>
       <tr>
-        <th>Secret</th>
-        <td>{{ annotator.secret }}</td>
+        <th>Login URL</th>
+        <td><a href="{{ login_link }}">{{ login_link }}</a></td>
```

Comment:
I think it might be confusing if for some reason you can only tell the judge the link through an analog way or if an admin tries to copy the link by highlighting the text and copying that. These cases are really unlikely, but I don't see a drawback of just leaving the full link there, so I think it would be better to leave it as it is.